app/services/players/transfers.py

```
@@ -26,6 +29,50 @@ def __post_init__(self) -> None:
         self.raise_exception_if_not_found(xpath=Players.Profile.NAME)
         self.transfer_history = self.make_request(url=self.URL_TRANSFERS.format(player_id=self.player_id))
 
+    def __clean_html_value(self, value: str) -> Tuple[str, str]:
+        """
+        Clean HTML tags from a string value and extract just the currency value or preserve special text values.
+        Also determine the transfer type based on the fee text.
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)